.env

```diff
@@ -23,8 +24,13 @@ APP_SECRET='s$cretf0rt3st'
 ###> symfony/lock ###
 # Choose one of the stores below
 # postgresql+advisory://db_user:db_password@localhost/db_name
-LOCK_DSN=semaphore
+LOCK_DSN=flock
```

Comment:
I personally would prefer semaphore but Symfony ships now with flocks (semaphor is I think linux only).

What should we do? https://github.com/symfony/recipes/pull/1142

Reply:
@Prokyonn Sadly this is not possible via env variable and think it make sense to have the LOCK_DSN as env variable as it is more a devops decision to decide which one to use.

Reply:
@alexander-schranz then I would go with `flock` and add an comment that `semaphore` would be superior in linux systems

Reply:
@Prokyonn added acomment please have a look at it.